content/en/blog/2024/prometheus-compatibility-survey/index.md

```diff
@@ -0,0 +1,147 @@
+the current state of compatibility between the projects, and try to understand
+user sentiment on a few of the remaining open questions. In particular, we
+wanted feedback on the translation for metric names, as we want to be sure
+before we make any changes.
+
+The survey received 86 responses, and contained many helpful pieces of feedback.
+Than you to everyone that participated!
+
+## Overall Takeaways
+
+- A slight majority (54%) prefer leaving the dots in the metric name, rather
+  than translating to underscores.
+- A slight majority (54%) prefer having the unit in the name, but only 37% think
+  it should be required.
+- Respondents who prefer units in the metric name are likely to also prefer
+  translating dots to underscores.
+- The best predictors of the "units and underscores" group are Prometheus server
+  experts and being an SRE.
+- The best predictors of the "no units and dots" group are OpenTelemetry library
+  experts and being a developer.
+
+## Who took the survey
+
+Survey respondents were mostly from large (>1000 employees) companies (52%) in
+the Technology industry (71%). Respondents were more likely to consider
+themselves experts with Prometheus-related topics than with
+OpenTelemetry-related topics, and were evenly distributed across roles. Nearly
+all respondents (>90%) stored metrics in the Prometheus server or another open
+source Prometheus backend, and nearly all use PromQL to query their metrics.
+
+## Sentiment on the Current State
+
+Overall, respondents were neutral on the question of whether OpenTelemetry was
+easy to use with Prometheus, and considered the current translation between
+OpenTelemetry and Prometheus somewhat confusing. This was consistent regardless
+of their opinions on units or delimiters.
+
+## Dots and Underscores
+
+OpenTelemetry [specifies](/docs/specs/semconv/general/attribute-naming/) that
+conventions should use dots as the namespace delimiter, and underscores as the
+delimiter between "multi-word-dot-delimited components" (e.g.
+`http.response.status_code`). On the other hand, Prometheus
+[uses underscores](https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels)
+as its delimiter.
+
+Currently, when exporting in Prometheus format from an OpenTelemetry SDK, all
+dots are changed to underscores to comply with the Prometheus requirements. We
+wanted to learn whether OpenTelemetry users who used these exporters preferred
+to keep the dots in the original metric name, or liked the consistency with
+existing Prometheus metrics of translating to underscores.
+
+Of users who indicated they used OpenTelemetry for metrics, and PromQL as their
+query language, 54% preferred keeping the original OpenTelemetry metric name
+including dots, and 46% want metric names that match Prometheus conventions with
+only underscores.
+
+![Dots vs underscores pie chart](dots-vs-underscores.png)
+
+When we asked about specific example PromQL queries or alerts, the results
```

Comment:
For the uninitiated, I'd be great to know:

- What are the difficulties that this difference posed so far.
- How is this being tackled, if at all.

Reply:
added to the background section